Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Builds in ~ 5 min and is ~ 2.8 GB on a linux laptop
+FROM mambaorg/micromamba:0.15.3
+
+# Label image following opencontainers image-spec annotations recommendation:
+# https://github.com/opencontainers/image-spec/blob/main/annotations.md
+LABEL org.opencontainers.image.description="Container for InSAR time series analysis with MintPy"
+LABEL org.opencontainers.image.authors="Forrest Williams <forrestfwilliams@icloud.com>, Joseph H Kennedy <me@jhkennedy.org>, Andre Theron <andretheronsa@gmail.com>"
+LABEL org.opencontainers.image.url="https://github.com/insarlab/MintPy"
+LABEL org.opencontainers.image.source="https://github.com/insarlab/MintPy"
+LABEL org.opencontainers.image.documentation="https://mintpy.readthedocs.io/en/latest/"
+LABEL org.opencontainers.image.licenses="GPL-3.0-or-later"
+
+# Dynamic lables to define at build time via `docker build --label`
+# LABEL org.opencontainers.image.created=""
+# LABEL org.opencontainers.image.version=""
+# LABEL org.opencontainers.image.revision=""
+
+USER root
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PYTHONDONTWRITEBYTECODE=true
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends git vim wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+USER micromamba
+WORKDIR /home/micromamba
+
+ENV PATH=/opt/conda/bin:${PATH}
+
+ARG MINTPY_HOME=/home/micromamba/tools/MintPy
+COPY --chown=micromamba:micromamba . ${MINTPY_HOME}/
+
+ARG PYTHON_VERSION="3.8"
+RUN micromamba install -y -n base -c conda-forge python=${PYTHON_VERSION}  \
+      jupyterlab ipympl -f ${MINTPY_HOME}/docs/requirements.txt && \
```

Comment:
We should add `gdal>=3` to the environment as well since it's required by so many upstream InSAR processors.